src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+const ViewDependentsListItem = props => (
+  <div className="vads-l-col--12 vads-u-background-color--gray-lightest vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-padding-top--1 vads-u-padding-bottom--2 vads-u-padding-x--2">
+    <div className="vads-l-row">
+      <div className="vads-l-col--9">
+        <p className="vads-u-font-weight--bold vads-u-margin-top--0p25 vads-u-margin-bottom--0 vads-u-margin-x--0 vads-u-font-size--lg">
+          Jesse Cohn
+        </p>
+      </div>
+      {props.spouse ? (
+        <div className="vads-l-col--3">
+          <p className="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
+            Spouse
+          </p>
+        </div>
+      ) : (
+        <div className="vads-l-col--3">
+          <p className="vads-u-font-weight--bold vads-u-font-size--lg vads-u-text-align--right vads-u-margin--0">
+            Child
+          </p>
+        </div>
+      )}
+    </div>
+    <div className="vads-l-row vads-u-margin-y--0p5">
+      <div className="vads-l-col--12 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
+        {props.onAward ? (
+          <p className="vads-u-margin-right--2 vads-u-margin-top--0">
+            <i className="fas fa-medal vads-u-margin-right--0p5" />
+            <span className="vads-u-font-weight--bold">On Award</span>{' '}
+          </p>
+        ) : null}
+
+        {props.social ? (
+          <p className="vads-u-margin-top--0 vads-u-margin-bottom--0">
+            SSN: <strong>{props.social}</strong>
+          </p>
+        ) : null}
+
+        {props.birthdate ? (
+          <p className="vads-u-margin-left--2 vads-u-margin-top--0 vads-u-margin-bottom--0">
+            Date of birth: <strong>{props.birthdate}</strong>
+          </p>
+        ) : null}
+
+        {props.age ? (
+          <p className="vads-u-margin-left--2 vads-u-margin-top--0 vads-u-margin-bottom--0">
+            Age: <strong>{props.age}</strong>
+          </p>
+        ) : null}
+      </div>
+    </div>
+    {props.changeStatus ? (
```

Comment:
Do we know yet if there's a scenario where changing the status of a dependent isn't an option? This doesn't seem like something that needs to be conditionally rendered.

Reply:
This is really just to hide the button till we have the rest of the functionality needed so that something is achievable by clicking the button. I can remove the button totally if thats better?

Reply:
Yeah I realized this during sprint planning this morning. It might be worth just removing for now since it seems this functionality is gonna be blocked for a while. I'll leave it up to you.